boards/crowboard/main.py

```diff
@@ -0,0 +1,93 @@
+import board
+
+from kb import KMKKeyboard
+
+from kmk.keys import KC
+from kmk.kmk_keyboard import KMKKeyboard
```

Comment:
This import is wrong. This overrides the import from above. Sorry that I missed this.